node/src/stream_handler_pool.rs

```diff
@@ -1137,6 +1137,7 @@ mod tests {
 
     #[test]
     fn stream_handler_pool_creates_nonexistent_stream_for_reading_and_writing() {
+        use crossbeam_channel::unbounded;
```

Comment:
Again...but off course, the previous new import will cover this one. Just remove this.